mlir/test/Dialect/MIOpen/lowering_output_tensor_kyxc_nhwc_nhwk.mlir

```diff
@@ -44,9 +44,9 @@ func @miopen_conv2d_bwd_data_gkyxc_nhwgc_nhwgk(%filter : memref<1x128x3x3x8xf32>
 // CHECK:       names = ["gemmG"]
 // CHECK:       source_names = ["go"]
 // CHECK:       names = ["gemmK"]
-// CHECK:       source_names = ["ko"]
+// CHECK:       source_names = ["ko", "ydotslice", "xdotslice"]
```

Comment:
Same comment as above

Reply:
Actually the original code is correct

Reply:
ko is k,  ydotslice is YDotslice and xdotslice is XDotSlice